DevGuide.md

```diff
@@ -1,12 +1,14 @@
 # Development guide for Pokete
+
 This guide only handles adding new types, attacks and Poketes to Pokete, for further information on class and function syntax see the html documentation or `$ pydoc <file>`.
 
 1. [Adding Poketes](#adding-poketes)
-2. [Adding attacks](#adding-attacks)
-3. [Adding types](#adding-types)
-4. [Adding subtypes](#adding-subtypes)
+1. [Adding attacks](#adding-attacks)
+1. [Adding types](#adding-types)
+1. [Adding subtypes](#adding-subtypes)
```

Comment:
Making pre-commit change markdown files completely kills the purpose of markdown, markdown should be readable by mashines and humans and for example replacing all numbers in a list with one dues not help that.
Also why doe it escape `[]`?

Reply:
I adopt `mdformat` for formatting markdown files, but it seems to have some errors. I replaced it with `markdownlint-fix`.

Reply:
I adopt `mdformat` for formatting markdown files, but it seems to do some problematic behavior. I dropped and replaced it with `markdownlint-fix`.